lib/WebService/Async/CustomerIO.pm

```diff
@@ -0,0 +1,313 @@
+package WebService::Async::CustomerIO;
+
+use strict;
+use warnings;
+
+our $VERSION = '0.001';
+
+=head1 NAME
+
+WebService::Async::CustomerIO - unofficial support for the Customer.io service
+
+=head1 SYNOPSIS
+
+=head1 DESCRIPTION
+
+=cut
+
+use parent qw(IO::Async::Notifier);
+
+use mro;
+use Syntax::Keyword::Try;
+use Future;
+use Net::Async::HTTP;
+use Carp qw();
+use JSON::MaybeUTF8 qw(:v1);
+
+use WebService::Async::CustomerIO::Customer;
+use WebService::Async::CustomerIO::RateLimiter;
+use WebService::Async::CustomerIO::Trigger;
+
+use constant {
+    TRACKING_END_POINT => 'https://track.customer.io/api/v1',
+    API_END_POINT      => 'https://api.customer.io/v1/api',
+    RPS_LIMIT_TRACKING => 30,
```

Comment:
Better to use descriptive name, I guess its `REQUEST_PER_SECOND`

Reply:
fixed